src/unit/format.js

```diff
@@ -0,0 +1,49 @@
+define([
+	"../common/format-message"
+], function( formatMessage ) {
+
+/**
+ * format( value, unit, pluralGenerator )
+ *
+ * @value [Number]
+ *
+ * @unitProperies [Object]: localized unit data from cldr.
+ *
+ * @pluralGenerator [Object]: A pluralGenerator from Globalize.pluralGenerator.
+ *
+ * TODO pass along numberFormatter
```

Comment:
Let me correct myself... The number needs to be formatted using `numberFormatter()` (instead of being passed directly to the formatMessage as is below) in this PR, but we can defer allowing users to pass a different `numberFormatter()` as an option to a subsequent PR.

Use [relativeTimeFormatter](https://github.com/jquery/globalize/blob/master/src/relative-time.js#L67) as reference. But, note that I suggested a minor different thing than what relativeTimeFormatter does. It passes along its own options to the numberFormatter, which raises a new point: both should behave analogously.

I believe allowing a user to pass the formatter as an option is better than mixing options of both unit formatter and number formatter. What do you think?